lib/options.json

```diff
@@ -288,6 +288,16 @@
     "setup": {
       "instanceof": "Function"
     },
+    "sockHost": {
+      "anyOf": [
+        {
+          "type": "string"
+        },
+        {
+          "type": "null"
```

Comment:
I think we don't have to write explicitly.

Reply:
You mean the `"type": "null"`? The `host` option has the same format.

Reply:
Yes, we discussed here, and I also don't think it's necessary.

fyi: https://github.com/webpack/webpack.js.org/pull/2995#discussion_r281509754